boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -16,14 +16,6 @@
 			#size-cells = <1>;
 			ranges = <0x0 0x2f010000 0x41000>;
```

Comment:
We can extend cpuapp_ram0x to the start of 0x2f000000 now. That includes deleting the offset for cpuapp_data defined still below